JavaAssignmentZemoso/Assignment10/Linkedlist.java

```diff
@@ -0,0 +1,125 @@
+/*
+@author:- Chiranjiv Goyal
+
+Create a generic, singly linked list class called SList, which, does not implement the List interface.
+    
+Each Link object in the list should contain a reference to the next element in the list, 
+but not the previous one (LinkedList, in contrast, is a doubly linked list, 
+which means it maintains links in both directions).
+
+Create your own SListIterator which, does not implement ListIterator. 
+The only method in SList other than toString( ) should be iterator( ), which produces an SListIterator.
+
+The only way to insert and remove elements from an SList is through SListIterator.
+
+*/
+
+package Assignment10;
+
+import java.util.*;
+
+class Slist 
+{
+    private LinkedListObject headlink = new LinkedListObject(null);
```

Comment:
Constructor could be overloaded instead of passing null. Passing null in codes must be avoided.